R/theme_element.R

```diff
@@ -1,23 +1,53 @@
 
-element_block <- function(background_fill,
-                          outline,
-                          post_space = c(NULL, " ", "", "------")){
+#' Element Row Group Location
+#'
+#' @param location Location of the grouping. Where 'indented' has all groups in
```

Comment:
#' @param location Location of the row group labels. Specifying 'indented' combines all group and label variables into a single column with each sub-group indented under its parent. 'spanning' and 'column'
#'  retain the highest level group variable in its own column and combine all remaining group and label variables into a single column with sub-groups indented. The highest level group column will either be printed as a spanning header or in its own column in the gt.
#' @param indent A string of the number of spaces you want to indent